tests/testthat/test-build_home.R

```diff
@@ -67,3 +67,30 @@ test_that("links to vignettes & figures tweaked", {
 
   expect_output_file(cat(as.character(html)), "home-links.html")
 })
+
+
+# cran --------------------------------------------------------------------
+
+
+test_that("package CRAN verification", {
+
+  #Package on CRAN, checking against default
+  expect_equal(on_cran("dplyr"), TRUE)
+  #Package on CRAN, checking against alternate mirror 1
+  expect_equal(on_cran("dplyr", "https://cran.stat.auckland.ac.nz"), TRUE)
+  #Package on CRAN, checking against alternate mirror 2
+  expect_equal(on_cran("dplyr", "https://cran.cnr.berkeley.edu"), TRUE)
+  #Package on CRAN, checking against alternate mirror 3
+  expect_equal(on_cran("dplyr", "https://cloud.r-project.org"), TRUE)
```

Comment:
I'd just use this mirror - the others are less reliable, and this tests that the complete code path works.